src/Microsoft.VisualStudio.VsInteractiveWindow/VsInteractiveWindow.cs

```diff
@@ -122,6 +127,13 @@ private void SubmissionBufferAdded(object sender, SubmissionBufferAddedEventArgs
             GetToolbarHost().ForceUpdateUI();
         }
 
+        private void OnFramePropertyChanged(object sender, PropertyChangedEventArgs eventArgs)
+        {
+            if (eventArgs.PropertyName.Equals("Title", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
@tmat, it might help (at least for VS2017) if the repo had the C# Code Style options listed in the .editorconfig file.